src/api/app/models/cloud/backend/upload_job.rb

```diff
@@ -42,6 +42,11 @@ def self.all(user)
         []
       end
 
+      def created
+        Time.at(xml_object.created.to_i).to_datetime
+      end
+      alias_method :created_at, :created
```

Comment:
Why don't you just overwrite created_at directly?

Reply:
I want to keep that it is possible to access the backend response directly and the backend responds with created.

Reply:
Would you mind adding a comment to the commit message explaining why the created method is added and why we don't overwrite created_at?

Reply:
Done!